libcontainer/configs/config.go

```diff
@@ -246,6 +239,109 @@ type Capabilities struct {
 	Ambient []string
 }
 
+//
+// FsEntry type.
+//
+
+type FsEntryKind uint32
+
+const (
+	InvalidFsKind FsEntryKind = iota
+	FileFsKind
+	DirFsKind
+	SoftlinkFsKind
+)
+
+type FsEntry struct {
+	Kind FsEntryKind
```

Comment:
Add a comment regarding difference between `path` and `dst`.

Reply:
Added a description for each field. "Dst" field meaning can be easily derived from that.